method.rb

```diff
@@ -0,0 +1,99 @@
+module Enumerable
+  def my_each
+    (0...length).each do |i|
+      yield self[i]
+    end
+    self
+  end
+
+  def my_each_with_index
+    (0...length).each do |i|
+      yield self[i], i
+    end
+    self
+  end
```

Comment:
- [ ] should have working `my_each` identical and returns the same thing as ruby's [`each`](https://ruby-doc.org/core-2.6.4/Array.html#method-i-each). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_each.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
- You are not supposed to use the original ruby methods but to create your own. In this case, you can use a while loop that finishes when you reached the last element of the enumerable.
- Your method is not returning an enumerator when no block is given.
- consider that not all enumerables are iterable like this. What if you apply this method to a range? This will affect all of the other methods where you use `my_each_with_index`